x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_execution_summary_and_chart.tsx

```diff
@@ -154,40 +169,37 @@ export const RuleExecutionSummaryAndChart = (props: RuleExecutionSummaryAndChart
           color={showDurationWarning ? 'warning' : 'subdued'}
           hasBorder={false}
         >
-          <EuiStat
-            data-test-subj="avgExecutionDurationStat"
-            titleSize="xs"
-            title={
-              <EuiFlexGroup gutterSize="xs">
-                {showDurationWarning && (
-                  <EuiFlexItem grow={false}>
-                    <EuiIconTip
-                      data-test-subj="ruleDurationWarning"
-                      anchorClassName="ruleDurationWarningIcon"
-                      type="warning"
-                      color="warning"
-                      content={i18n.translate(
-                        'xpack.triggersActionsUI.sections.ruleDetails.alertsList.ruleTypeExcessDurationMessage',
-                        {
-                          defaultMessage: `Duration exceeds the rule's expected run time.`,
-                        }
-                      )}
-                      position="top"
-                    />
-                  </EuiFlexItem>
-                )}
-                <EuiFlexItem grow={false} data-test-subj="ruleEventLogListAvgDuration">
-                  {formatMillisForDisplay(computedRuleSummary.executionDuration.average)}
-                </EuiFlexItem>
-              </EuiFlexGroup>
-            }
-            description={i18n.translate(
+          <EuiText size="s" color="text">
+            {i18n.translate(
               'xpack.triggersActionsUI.sections.ruleDetails.alertsList.avgDurationDescription',
               {
                 defaultMessage: `Average duration`,
               }
             )}
-          />
+          </EuiText>
+          <EuiFlexGroup gutterSize="xs">
+            {showDurationWarning && (
+              <EuiFlexItem grow={false} style={{ paddingTop: 4 }}>
```

Comment:
as we are using flexbox, what about setting `alignItems="center"` in the flex group? I think you are adding the padding top to align the icon and text, which should be the same. Would be nice to get rid of this "magic" value of 4

Reply:
Haha, good one!
I am not an expert in this topic, I will give your suggestion a try, your approach makes more sense 👍🏻